code/src/OSFramework/Feature/IDirtyMark.ts

```diff
@@ -3,7 +3,7 @@ namespace OSFramework.Feature {
     export interface IDirtyMark {
         clear(): void;
         // eslint-disable-next-line @typescript-eslint/no-explicit-any
+        clearPropertyInRow(row: any): void;
```

Comment:
Can you please add  a comment here? We should have comments all over the interfaces to make it easier for the other developers.